src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,87 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private final File file;
+
+    /**
+     * Creates a new Storage file where tasks can be loaded from and saved to.
+     * @param filepath represents the path of the file.
```

Comment:
I think there is an empty line between description and parameters :"( for other javadoc comments also!